src/coreclr/vm/prestub.cpp

```diff
@@ -2843,7 +2843,6 @@ void ProcessDynamicDictionaryLookup(TransitionBlock *           pTransitionBlock
 
     TADDR genericContextPtr = *(TADDR*)GetFirstArgumentRegisterValuePtr(pTransitionBlock);
 
-    pResult->testForFixup = pResult->testForNull = false;
```

Comment:
Shouldn't we keep the `pResult->testForNull = false;` part?

Reply:
Good catch, since the caller doesn't zero initialize the struct we need to init all fields